File: fibonacci/fibonacci.js
/**
 * @description Calculates the Fibonacci sequence using recursion
 *
 * @param {number} n Fibonacci number index to calculate
 * @returns {number} N-th Fibonacci number
 */
function fibonacciJS(n) {
  if (n <= 1) {
    return n;
  } else {
    return fibonacciJS(n - 1) + fibonacciJS(n - 2);
  }
}

File: fibonacci/fibonacci.c
/**
 * @brief Calculates the Fibonacci sequence using recursion
 *
 * @param n The index of the Fibonacci number to calculate
 * @return N-th Fibonacci number
 */
int fibonacciC(int n) {
  if (n <= 1) {
    return n;
  } else {
    return fibonacciC(n - 1) + fibonacciC(n - 2);
  }
}
